docs/developer/app-store/apps/stripe/overview.mdx

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_label: Overview
+sidebar_position: 1
+title: Stripe App Overview
+---
+
+import { AppMetadata } from "/components/AppMetadata/AppMetadata.jsx";
+
+<AppMetadata
+  minSaleorVersion="3.21"
+  githubUrl="https://github.com/saleor/apps/tree/main/apps/stripe"
+/>
+
+Saleor Stripe App is a payment gateway integration that allows customers to pay for their checkouts/orders with Stripe.
+
+Stripe is a leading payments platform offering multiple solutions for millions of companies.
+
+:::note
+App is in Preview Mode, which means it may change significantly. Saleor team will not introduce a breaking change, but it may occur that new app version will be released in short time period and the Preview version will not be supported.
+:::
+
+## Features
+
+The Stripe app offers end-to-end payment flow integration using Transactions API on the Saleor side and Payment Intents API on the Stripe side.
+
+The app:
+
+- Allows seamless configuration using Saleor Dashboard UI
+- Creates Stripe webhook automatically when configuration is created (and removes webhook when configuration is deleted)
+- Allows to create multiple configurations (like production/testing or per-channel environments) and map them to specific Saleor channels
+- Supports manual capture (authorization flow) and instant charge flow
```

Comment:
I don't know what nomeclature we want to use but in Stripe is is called: Place a hold on a payment method

https://docs.stripe.com/payments/place-a-hold-on-a-payment-method